src-tauri/tauri.conf.json

```diff
@@ -18,13 +18,13 @@
                 "sidecar": true,
                 "scope": [
                     {
-                        "name": "pkexec",
-                        "cmd": "pkexec",
+                        "name": "arpaname",
+                        "cmd": "arpaname",
                         "args": true
                     },
                     {
-                        "name": "which",
```

Comment:
Why did you remove the which command? Please leave because I am not sure what other tools require that command.